pypeit/core/coadd.py

```diff
@@ -92,8 +99,10 @@ def new_wave_grid(waves, wave_method='iref', iref=0, A_pix=None, v_pix=None, **k
         if A_pix is None:
             msgs.error("Need to provide pixel size with A_pix for with this method")
         #
-        wave_grid_min = np.min(waves)
-        wave_grid_max = np.max(waves)
+        if wave_grid_min is None:
```

Comment:
@feigewang?

Reply:
@profxj @jhennawi Basically, we will try to use one code to coadd everything (both individual exposures and echelle orders). Adding this will allow user to give the wave_grid_min and wave_grid_max which is needed for coadding different echelle orders. default None is the same as previous version (using the min and max wavelength of the iref spectrum).

Reply:
This stuff is still in development.